### ISSUE TITLE
Add Cash By Mail payment method?

### ISSUE THREAD

**Issue by crocket** [btdex/btdex#137]
I don't know whether individual fiat currency payment methods should be added.

**Comment by boner-cmd** [comment]
### Legal concerns

In the United States, there are anti-money-laundering risks to doing so, at the very least. Payment disputes in btdex are relatively low risk, with limited recourse. If you take payment by mail and there's a dispute, you could open yourself up to mail fraud concerns, which has a maximum prison sentence of _20 years_, not to mention the fine.

### Non-repudiation

Also, an important part of btdex is that the offer is **not** based on time of receipt. The handling delay and resulting slippage is something you should price into the transaction, and if the trade is no longer valid, you should be willing to lose your deposit. If you are on the Signum side of the transaction, you can expect to lose your deposit and then some if the other person already sent the money. As an example, if I make an offer on btdex and the price moves against me, that person just made a profit. If the price moves against them, I just made a profit. _Profit is the trade incentive_. Nobody gets to unilaterally move the exchange rate just because they made a bad trade. A huge part of crypto is non-repudiation. Agreeing to buy or sell Signa at a set price and then reneging after the fact is just a scam.

### Verification

A bigger risk to fiat trades (if anything can be bigger than the risk of potentially going to court for criminal and civil charges) is that even without moving the exchange rate, they're very hard to verify. If I'm a mediator, I can confirm that you sent something on the blockchain and it was received by the correct address, but if I want to confirm that I sent cash, whether it was a money order, a check, or cash itself, there isn't a good way to do that. I can confirm that someone received the letter using the post office, but providing that verification is just a picture that could possibly be faked. I can't even confirm that the right person received the money-

**I've had delivery drivers deliver expensive, signature-secured corporate goods to a neighbor who even signed their own name 
 (meaning, not my name) on the receipt.**

So I can _kind of_ independently verify that _someone_ received the money, but that's really not good enough. 

Plus, once people have to start providing all these documents or images to prove a transaction occurred, there's a lot of personal information involved (names, mailing addresses, photos, records). Another big advantage of btdex is the lack of KYC. When I use a major exchange, I have to give someone, sometimes not even the exchange, a lot of personal information for KYC. The theory is that I'm giving a trusted central authority that info. Trying to verify cash by mail would involve giving that info for every disputed trade to mediators or the other party as part of a dispute; it's pretty much all the downsides of centralization and KYC, but worse in every way.

There's also the high non-trade costs in terms of tracking registration, insurance, special packaging, even potentially image quality requirements. 

### At this point, anyone who wants to do CBM might as well use a traditional exchange. They'll disclose less information, have a lower regulatory burden (only verifying once, not per transaction), spend less time preparing, complete faster, and it will cost less out of pocket.

As for how [Bisq ](https://bisq.wiki/Cash_by_mail) does it, in addition to the above, there are a lot of things to object to that are unique to Bisq:

- In the transaction that is the hardest to verify ("Cash-by-mail trades are less verifiable than other fiat trades. This makes handling disputes much harder") and requires the most trust ("the onus to act honestly is squarely on both peers"), there is no guaranteed recourse ("Mediators ... are NOT guaranteed to help." emphasis in original)
- Bisq allows adding additional terms to each transaction, making dispute resolution more complex and difficult in a transaction that is already more complex and difficult
- If there is a dispute that can't be resolved, Bisq will just... take the money from BOTH parties ("If a mediator is engaged, and if either peer rejects the mediator's suggestion, both peers' funds will be sent to a Bisq 'donation' address, and the trade will effectively be completed.")?

This is disastrous in every way. 

- If I send you the cash and you never confirm it, Bisq will take _your_ money, so they now have a financial incentive to **not** mediate. You and I both lose our money. Bisq gets a 2x transaction payout.
- If I never send you the money, Bisq can take your money anyway. Bisq gets a 1x transaction payout, and I'm not punished for not sending the cash. Which leads us to...
- Bisq says it will take both sides of the transaction, but it's cash by mail. How can Bisq take money from the cash side? Is there a Bisq-deposited account? If so, then if I send you the money and you don't confirm it, I lose _**double**_ the cost of the trade, and Bisq takes your money. So now Bisq has an incentive _**worth three times the value of the CBM trade**_ to not help either side, and the person who did the right thing (the cash side) is punished _**twice as much**_ as the person who didn't honor the agreement.

No thanks.

**Comment by crocket** [comment]
Do you know any better alternative?

**Comment by boner-cmd** [comment]
No. A system does not exist that will allow you to **safely** and **legally** avoid KYC. If you do _not_ have a KYC requirement, you can use any exchange you want. If your country has a KYC requirement for AML, you might as well just trade fiat to an arbitrary person who can trade it for crypto. The problem is, there is no real way to guarantee that transaction- you are putting absolute trust in a random person. It's definitely not safe.

That's essentially what is happening in Nigeria's underworld; sure, you can trade fiat for BTC, but if person you have to deal with is a criminal, and that means corruption and no recourse if you are defrauded.

**Comment by crocket** [comment]
> A system does not exist that will allow you to safely and legally avoid KYC.

* Localmonero uses reputation system to hold cash-by-mail traders accountable. Thus, it is arguably a better system for CBM than BTDex or Bisq.
* You haven't mentioned bank tracking at all. If you are willing to accept a degree of tracking, bank transfer deals without KYC are also possible. Bank account itself requires KYC in advance, but you don't have to state the purpose for transfer. Thus, you can buy crypto without letting banks know that you bought crypto for now.
* I think the system does not exist, yet. But, it can exist in the near future. There's huge pressure to avoid any form of KYC.

**Comment by boner-cmd** [comment]
To be clear, your examples of how a safe system without KYC does exist are:

1) A rep system, which is a form of trusting an arbitrary stranger, which I already said.

2) "Bank tracking," which is a form of KYC that, if you are okay with, you can just do through a KYC exchange, which I already said.

3) Admitting it doesn't exist, which I already said.

So yes, I agree.

But, since you say that Localmonero may be better than btdex, you answered your own question about alternatives.

But a btdex ticket is not a place for debate. I gave my reasons not to add a "feature." That's all.

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by crocket** [comment]
I don't think localmonero is necessarily going to stay as a better alternative in the long term. Localmonero may succumb to KYC pressure in the future because it is a centralized exchange.
